tests/test_bucketlist_resource.py

```diff
@@ -152,7 +212,7 @@ def test_cant_create_bucketlist_without_authentication(self):
             '/api/v1/bucketlists',
             data=json.dumps(new_bucketlist),
             headers=no_token)
-        self.assertTrue(response.status_code==401)
+        self.assertTrue(response.status_code == 401)
```

Comment:
You can also use `self.assertEqual(response.status_code==401)` for testing equality